### PR TITLE
Add classes/ids to some controls for styling

### PR DESCRIPTION
**Pull request by DrJKL**

### CODE REVIEW

File: scripts/controlnet.py [hunk 1]
```diff
@@ -355,10 +355,10 @@ def refresh_all_models(*inputs):
         if shared.opts.data.get("controlnet_disable_control_type", False):
             type_filter = None
         else:
-            with gr.Row():
-                type_filter = gr.Radio(list(preprocessor_filters.keys()), label=f"Control Type", value='All', elem_id=f'{elem_id_tabname}_{tabname}_controlnet_type_filter_radio')
+            with gr.Row(elem_id="controlnet_control_type"):
```
Comment (huchenlei) [inline scripts/controlnet.py:358]: I think use `elem_id` here will cause element id conflict among multiple controlnet ui-groups. Please replace `elem_id` with `elem_classes`. Same in line 361.

Comment (DrJKL) [reply scripts/controlnet.py]: Okay, classes it is.

**Comment by DrJKL** [comment]
For context, this sets up for this other PR: https://github.com/vladmandic/automatic/pull/1187

**Comment by lllyasviel** [comment]
why we remove text "[Loopback]"? this may influence some language packages that match gradio texts

**Comment by DrJKL** [comment]
The Loopback is still present, but as a label
image

**Comment by lllyasviel** [comment]
yes I believe they match full string (at HTML level)

**Comment by DrJKL** [comment]
Okay, reverted the label/info. I do think it'd be nicer to have a shorter name there. Are the localization files in their own repo?

**Comment by lllyasviel** [comment]
ok merged

**Pull request merged**